qa/integration-tests/src/test/java/io/camunda/it/client/SearchDecisionRequirementsTest.java

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.client;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.qa.util.cluster.TestStandaloneCamunda;
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+@ZeebeIntegration
+public class SearchDecisionRequirementsTest {
+  private static Long decisionRequirementKey;
+  private static String decisionRequirementId;
+  private static String decisionRequirementName;
+
+  @TestZeebe
+  private static TestStandaloneCamunda testStandaloneCamunda = new TestStandaloneCamunda();
+
+  private static ZeebeClient zeebeClient;
+
+  @BeforeAll
+  public static void setup() {
+    zeebeClient = testStandaloneCamunda.newClientBuilder().build();
+
+    deployProcessModel("decision/decision_model.dmn");
```

Comment:
it seems there is no easy way to programmatically build a Decision to deploy like we do for processes, rather than using external resource.